includes/TinyMCE.php

```diff
@@ -59,7 +59,9 @@ public function init() {
 	 * @return array
 	 */
 	public function tinymce_web_stories_button( array $buttons ) {
-		array_push( $buttons, 'web_stories' );
+		if ( ! $this->is_block_editor() ) {
```

Comment:
Instead of having this in every function, why not just add in the init function.

```
	public function init() {
               if ( $this->is_block_editor() ) {
                     return;
               }
```

Reply:
`init` runs too early and `$current_screen` is not set there yet, hence put it for the individual actions ensuring that $current_screen will be there.

Reply:
init is loaded on `init` action. But there is no reason, we could load this on a different action, like `admin_init` or another action where screen is setup.  Doing this in 4 places isn't very [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

Reply:
We could even use `current_screen` action.